Source/PeleLM.cpp

```diff
@@ -1861,75 +1861,35 @@ PeleLM::initData ()
   if (verbose) amrex::Print() << "initData: finished init from pltfile" << '\n';
 #endif
 
-
-#if 1
-
-
-    const auto geomdata = geom.data();
+  const auto geomdata = geom.data();
 
 #ifdef _OPENMP
 #pragma omp parallel if (Gpu::notInLaunchRegion())
 #endif
-    for (MFIter mfi(S_new); mfi.isValid(); ++mfi)
-    {
-        const Box& box = mfi.validbox();
-        auto sfab = S_new.array(mfi);
-        auto pressfab = P_new.array(mfi);
-
-        amrex::ParallelFor(box,
-        [=] AMREX_GPU_DEVICE (int i, int j, int k) noexcept
-        {
-            pelelm_initdata(i, j, k, sfab, pressfab, geomdata);
-        });
-    }
-
-
-
-#else
-
-
-
-
-
-#ifdef _OPENMP
-#pragma omp parallel
-#endif
-  for (MFIter snewmfi(S_new,true); snewmfi.isValid(); ++snewmfi)
+  for (MFIter mfi(S_new); mfi.isValid(); ++mfi)
```

Comment:
Needs tilingIfNotGpu() argument (until that becomes the default, as it should be)